java/parser/test/ParserTest.java

```diff
@@ -1013,16 +1003,6 @@ public void testDefineRules() {
         assertQueryEquals(expected, parsed, query.replace("'", "\""));
     }
 
-    @Test
-    public void testQueryParserWithoutGraph() {
-        final String queryString = "match\n" +
-                "$x isa movie;\n" +
-                "get $x;";
-        TypeQLMatch query = parseQuery("match\n" +
-                                               "$x isa movie; get $x;").asMatch();
-        assertEquals(queryString, query.toString());
-    }
-
```

Comment:
Almost identical to the very first `testSimpleQuery` test, but doesn't compare with a builder result.